docs/STACK-DESIGN.md

```
@@ -293,23 +303,23 @@ $ josh-filter ':/live-markdown' --update refs/heads/extracted HEAD
 3c76f5629d1c7cb78d51cd4d8cf36d9c6c1bf42f      <- the recorded cursor, byte for byte
 ```
 
-Filtering a member carrying one workspace commit continues upstream's history
+Filtering a member carrying one stackspace commit continues upstream's history
 rather than restating it:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)